doc/src/Howto_pylammps.rst

```diff
@@ -1,564 +1,6 @@
-moves random atoms by a random delta and only accepts certain moves.
-
-.. code-block:: python
-
-   estart = L.eval("pe")
-   elast = estart
-
-   naccept = 0
-   energies = [estart]
-
-   niterations = 3000
-   deltamove = 0.1
-   kT = 0.05
-
-   natoms = L.system.natoms
-
-   for i in range(niterations):
-       iatom = random.randrange(0, natoms)
-       current_atom = L.atoms[iatom]
-
-       x0, y0 = current_atom.position
-
-       dx = deltamove \* random.uniform(-1, 1)
-       dy = deltamove \* random.uniform(-1, 1)
-
-       current_atom.position = (x0+dx, y0+dy)
-
-       L.run(1, "pre no post no")
-
-       e = L.eval("pe")
-       energies.append(e)
-
-       if e <= elast:
-           naccept += 1
-           elast = e
-       elif random.random() <= math.exp(natoms\*(elast-e)/kT):
-           naccept += 1
-           elast = e
-       else:
-           current_atom.position = (x0, y0)
-
-The energies of each iteration are collected in a Python list and finally plotted using matplotlib.
-
-.. image:: JPG/pylammps_mc_energies_plot.jpg
-   :align: center
-
-The IPython notebook also shows how to use dump commands and embed video files
-inside of the IPython notebook.
-
-Using PyLammps and mpi4py (Experimental)
-----------------------------------------
-
-PyLammps can be run in parallel using `mpi4py
-<https://mpi4py.readthedocs.io>`_. This python package can be installed
-using
-
-.. code-block:: bash
-
-   pip install mpi4py
-
-.. warning::
-
-   Usually, any :py:class:`PyLammps <lammps.PyLammps>` command must be
-   executed by *all* MPI processes. However, evaluations and querying
-   the system state is only available on MPI rank 0.  Using these
-   functions from other MPI ranks will raise an exception.
-
-The following is a short example which reads in an existing LAMMPS input
-file and executes it in parallel.  You can find in.melt in the
-examples/melt folder.  Please take note that the
-:py:meth:`PyLammps.eval() <lammps.PyLammps.eval>` is called only from
-MPI rank 0.
-
-.. code-block:: python
-
-   from mpi4py import MPI
-   from lammps import PyLammps
-
-   L = PyLammps()
-   L.file("in.melt")
-
-   if MPI.COMM_WORLD.rank == 0:
-       print("Potential energy: ", L.eval("pe"))
-
-   MPI.Finalize()
-
-To run this script (melt.py) in parallel using 4 MPI processes we invoke the
-following mpirun command:
-
-.. code-block:: bash
-
-   mpirun -np 4 python melt.py
-
-Feedback and Contributing
--------------------------
-
-If you find this Python interface useful, please feel free to provide feedback
-and ideas on how to improve it to Richard Berger (richard.berger@outlook.com). We also
-want to encourage people to write tutorial style IPython notebooks showcasing LAMMPS usage
-and maybe their latest research results.
+The PyLammps interface is deprecated and will be removed in a future release of
+LAMMPS. As such, the PyLammps version of this tutorial has been removed and is
+replaced by the :doc:`Python_head`.
```

Comment:
we could add the version of the last LAMMPS release that supports PyLammps here as well (presumably the last stable release).

Reply:
@megmcca please note the comment `will be removed in a future release of LAMMPS`, so it is still functional. In fact, all the unit tests pass and none has been removed, yet. I expect we will pull the plug *after* the *next* stable release where a bunch of other disruptive changes will take place (see #4344). The Summer 2025 stable release will become another "important legacy" release https://www.lammps.org/download.html